model/event/event_finder.go

```diff
@@ -127,6 +127,16 @@ func ByGuid(guid string) db.Q {
 	})
 }
 
+func AdminEventsBefore(before time.Time, n int) db.Q {
+	return db.Query(bson.M{
+		DataKey + "." + resourceTypeKey: ResourceTypeAdmin,
+		ResourceIdKey:                   "",
+		TimestampKey: bson.M{
+			"$lt": before,
+		},
+	}).Sort([]string{"-" + TimestampKey}).Limit(n)
```

Comment:
The index is { "r_id" : 1, "data.r_type" : 1, "ts" : 1 }}, will this be able to use the index?

Reply:
I'm going to add an index: https://jira.mongodb.org/browse/MCI-2768?filter=-1

Reply:
Can you add the index to scripts/indexes.js?

Reply:
oops I didn't see what query this was for - the highlighted query has sufficient index coverage by the existing one. I pushed a change to add the index for the query above this one